### ISSUE TITLE
Expokit 32 - FBAudienceNetwork 4.99 out of date.

### ISSUE THREAD

**Issue by owenhope** [expo/expo#3309]
Hello,

I need to update FBAudienceNetwork to 5.+ for iOS ... I managed to do it via gradle for android but cannot figure it out for iOS.

The error states:

```
FBAdErrorDetailKey = {
    msg = "The SDK version is in the ad request is no longer supported for new apps. Please upgrade to one of the latest versions of the SDK
}
```

My Podfile.lock 

```
- ExpoKit/Core (32.0.0):
    - Amplitude-iOS (~> 3.8)
    - Analytics (~> 3.5)
    - AppAuth (~> 0.4)
    - Branch (~> 0.24.2)
    - CocoaLumberjack (~> 3.2.1)
    - Crashlytics (~> 3.8)
    - EXCameraInterface
    - EXConstantsInterface
    - EXCore
    - EXFileSystemInterface
    - EXPermissionsInterface
    - EXReactNativeAdapter
    - EXSensorsInterface
    - Fabric (~> 1.6)
    - FBAudienceNetwork (= 4.99.0)
    - FBSDKCoreKit (= 4.38.1)
    - FBSDKLoginKit (= 4.38.1)
    - FBSDKShareKit (= 4.38.1)
    - Google-Maps-iOS-Utils (~> 2.1.0)
    - Google-Mobile-Ads-SDK (~> 7.22.0)
    - GoogleMaps (~> 2.5.0)
    - GoogleSignIn (~> 4.1)
    - JKBigInteger2 (= 0.0.5)
    - lottie-ios (~> 2.5.0)
    - React
```

Any help would be much appreciated.

**Comment by owenhope** [comment]
Thanks for the tag @AdamJNavarro ... 

I tried to just load a local version of the Podfile with a modification to the FBAudienceNetwork but I run into some other errors once the app loads.

Is there a way I can temporarily fix this issue (updating FBAudienceNetwork to 5.1.0) in the Podfile to override what is in the Expokit Podspec? Everytime I try it just says the dependencies don't match.

Pretty big blocker for us, as our app isn't showing ads.

**Comment by AdamJNavarro** [comment]
Hey @owenhope,

I relayed this to some of the engineers who usually end up updating our dependencies for ExpoKit. Hopefully we can get more information to you about this soon.

Cheers,

Adam

**Comment by c00piser** [comment]
I'm having this problem too. Any updates on this ?

**Comment by EvanBacon** [comment]
Closing as ExpoKit is now [deprecated in SDK 37](https://blog.expo.io/expo-sdk-37-is-now-available-dd5770f066a6). Please use the bare workflow instead.
